### PR TITLE
APEXMALHAR-2094 Quantiles Operator

### PR DESCRIPTION
**Pull request by sandeep-n**
[Opening new PR since PR #286 closed itself when I squashed commits.]
Addressing JIRA issue https://issues.apache.org/jira/browse/APEXMALHAR-2094. 
Added a new module "sketches" as per https://issues.apache.org/jira/browse/APEXMALHAR-2094?focusedCommentId=15296761. The sketches module will hold sketching algorithms including quantiles sketch.

### CODE REVIEW

File: sketches/src/main/java/org/apache/apex/malhar/sketches/QuantilesEstimator.java [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+   */
+  public long streamLength() {
+    return quantilesSketch.getN();
+  }
+
+  /**
+   * This field determines the specific quantiles to be calculated.
+   * Default is set to compute the standard quartiles.
+   */
+  private double[] fractions = {0.0, 0.25, 0.50, 0.75, 1.00};
```
Comment (ilganeli) [inline sketches/src/main/java/org/apache/apex/malhar/sketches/QuantilesEstimator.java]: move `fractions` and `pmfIntervals` up to where the other class variables are defined.

File: sketches/src/main/java/org/apache/apex/malhar/sketches/QuantilesEstimator.java [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+  /**
+   * This field determines the intervals on which the probability mass function is computed.
+   */
+  private double[] pmfIntervals = {};
+
+  /**
+   * Input port takes a number
```
Comment (ilganeli) [inline sketches/src/main/java/org/apache/apex/malhar/sketches/QuantilesEstimator.java]: This comment is not helpful. Input port also does not take a number, it takes a double.

File: sketches/src/main/java/org/apache/apex/malhar/sketches/QuantilesEstimator.java (hunk 1 above)
Comment (sandeep-n) [inline sketches/src/main/java/org/apache/apex/malhar/sketches/QuantilesEstimator.java]: moved

File: sketches/src/main/java/org/apache/apex/malhar/sketches/QuantilesEstimator.java (hunk 2 above)
Comment (sandeep-n) [inline sketches/src/main/java/org/apache/apex/malhar/sketches/QuantilesEstimator.java]: Removed redundant comment.

**Comment by gauravgopi123** [comment]
squash the commits?

**Pull request closed without merging**